### PR TITLE
Cran

### PR DESCRIPTION
**Pull request by kar-agg-gen**

### CODE REVIEW

File: tests/testthat/test-ui-heatmap.R [hunk 1]
```diff
@@ -42,6 +42,8 @@ test_that("cXheatmap9", {
 
 test_that("cXheatmap10", {
     check_ui_test(cXheatmap10())
+
+    message('The plot points fade on clicking the legends - working as designed')
```
Comment (cb4ds) [inline tests/testthat/test-ui-heatmap.R:46]: I don't think that this is specific enough to explain what we are making sure the tester understands is working as designed.  How about 'Plot points all fade on selecting shapeBy and sizeBy legend items - working as designed.'

File: tests/testthat/test-other-color-setting.R [hunk 2]
```diff
@@ -57,7 +57,7 @@ test_that("boxplot ColorScheme", {
                            groupingFactors          = list("dose"),
                            smpTitleScaleFontFactor  = 1.8,
                            stringSampleFactors      = list("dose"),
-                           title                    = "Boxplot - ColorScheme: Set1"
+                           title                    = "Boxplot - ColorScheme: Set2"
```
Comment (cb4ds) [inline tests/testthat/test-other-color-setting.R:60]: Any reason we changed this?

File: tests/testthat/test-ui-heatmap.R (hunk 1 above)
Comment (kar-agg-gen) [reply tests/testthat/test-ui-heatmap.R:46]: done

File: tests/testthat/test-other-color-setting.R (hunk 2 above)
Comment (kar-agg-gen) [reply tests/testthat/test-other-color-setting.R:60]: while testing I found out title was wrong ,,
colorScheme was set to "Set2",

**Pull request merged**